sdk/textanalytics/Azure.AI.TextAnalytics/tests/samples/SampleSnippets.cs

```diff
@@ -4,17 +4,14 @@
 using System;
 using System.Collections.Generic;
 using System.Threading.Tasks;
-using Azure.AI.TextAnalytics.Tests;
-using Azure.Core.TestFramework;
-using Azure.Identity;
 using NUnit.Framework;
 
 namespace Azure.AI.TextAnalytics.Samples
 {
     /// <summary>
     /// Samples that are used in the associated README.md file.
     /// </summary>
-    public partial class Snippets: SamplesBase<TextAnalyticsTestEnvironment>
+    public partial class Snippets : TextAnalyticsSampleBase
     {
         [Test]
         public void CreateTextAnalyticsClient()
```

Comment:
This looks like the only place that gets referenced in all samples for creating the client.